src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1205,6 +1206,19 @@ public void SetNameFromNodeNameAttribute()
 
         }
 
+        /// <summary>
+        /// Logs a user defined messages to the Dynamo Logger, this notifiaction will be dispalyed
+        /// in the Notification Extension.
+        /// </summary>
+        /// <param name="dynamoModel"></param>
+        /// <param name="shortMessage"></param>
+        /// <param name="detailedMessage"></param>
+        /// <param name="title"></param>
+        public void Notification(DynamoModel dynamoModel, string shortMessage, string detailedMessage, string title = "Notification")
```

Comment:
Maybe `Internal` for now and make it a verb like `LogNotification`. Our team usually keep the new APIs internal and need to discuss when to expose them.

Reply:
think we can avoid adding this altogether.

Reply:
why would we avoid it? as discussed on slack we are adding an API to the nodemodel to log notifications?